robot/Scenario-0.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Library           KubeLibrary
+
+*** Variables ***
+${NAMESPACE}  open5gs
+${POD_NAME}  .*gnb-ues*
+@{COMMAND_CHECK_PDU}  nr-cli  imsi-999700000000001  -e  status
+@{COMMAND_SEND_PING}  ping  -c1  -W1  -I  uesimtun0  172.17.0.1
+
+*** Test Cases ***
+Register UE
```

Comment:
Test case is actually only one: PDU session is up and running verification.
So I propose to transform current test cases into keywords and use them in mentioned test case.